File: level1/정수_제곱근_판별.cpp
#include <string>
#include <vector>
#include <cmath>

using namespace std;

long long solution(long long n) {
    
    if(pow((long long)sqrt(n), 2) != n)
        return -1;
    
    return pow((long long)sqrt(n)+1, 2);
}

File: JAVA/level1/정수_제곱근_판별.java
import java.lang.*;

class Solution {
    public long solution(long n) {
        long answer = 0;

        if(n == (long)(Math.pow((long)Math.sqrt(n), 2)))
           return (long)Math.pow(((long)Math.sqrt(n)+1), 2);
        return -1;
    }
}
